cmd/charmd/main.go

```diff
@@ -87,6 +87,18 @@ func serve(confPath string) error {
 	ring := bakery.NewPublicKeyRing()
 	ring.AddPublicKeyForLocation(cfg.IdentityLocation, false, &identityPublicKey)
 	cfg.PublicKeyLocator = ring
+	if conf.IdentityAPIPrivateKey != "" && conf.IdentityAPIPublicKey != "" {
```

Comment:
This seems like it would be better done in the config package. We should probably implement GetYAML and SetYAML methods on bakery.Key too to avoid this kind of boilerplate.